zstacklib/zstacklib/utils/ovn.py

```
@@ -61,10 +61,99 @@ def getAllDpdkNic():
+    for nicName, pciAddress in nicNamePciAddressMap.__dict__.items():
+        found = False
+        driver = ""
+        for dpdkNic in dpdkNics:
+            if dpdkNic.pciAddress == pciAddress:
+                found = True
+                driver = dpdkNic.driver
+                targetDpdkNic.append(dpdkNic)
+                break
+
+        if not found:
+            return 1, "nic [pci address: %s] is not found by dpdk-devbind.py".format(pciAddress)
```

Ruff (0.8.2)

`.format` call has unused arguments at position(s): 0

Remove extra positional arguments at position(s): 0

(F523)